package.json

```diff
@@ -42,10 +42,11 @@
   },
   "scripts": {
     "test": "npm run lint && npm run flow && jest --coverage",
+    "clean": "rimraf dist",
     "prepare": "npm run build",
     "prepack": "npm run prepareConfig:publish",
     "postinstall": "node ./helpers/check-version.js",
-    "build": "npm run prepareConfig:local && babel src -d dist",
+    "build": "npm run clean && npm run prepareConfig:local && babel src -d dist",
```

Comment:
I wonder if we should just do this in `prepare` only to avoid slowing down local development (i.e. `"prepare": "npm run clean && npm run build"`).

Reply:
Fair point, I can pop it in prepare instead. 

I had been debating on doing that, or the build step. Ultimately decided on the build step so it would be constantly cleaned + built.

Reply:
Moved it over to prepare instead